{"name": "Логин несуществующего пользователя: 404 и сообщение об ошибке", "status": "passed", "steps": [{"name": "Пытаемся залогиниться с несуществующими учетными данными", "status": "passed", "attachments": [{"name": "response.json", "source": "a6ed090a-2cac-4a22-b309-bebedc976bd0-attachment.json", "type": "application/json"}], "start": 1760894754778, "stop": 1760894754950}, {"name": "Проверяем 404 и текст ошибки", "status": "passed", "start": 1760894754950, "stop": 1760894754950}], "start": 1760894754778, "stop": 1760894754950, "uuid": "36f7871a-cf71-4856-ad96-13fb9127bf07", "historyId": "e4e1a5d5dd68c2cbdeede2f5a294d6e4", "testCaseId": "e4e1a5d5dd68c2cbdeede2f5a294d6e4", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_nonexistent_user_returns_error", "labels": [{"name": "feature", "value": "Courier"}, {"name": "epic", "value": "Scooter API"}, {"name": "story", "value": "Login courier"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "m00office0012.local"}, {"name": "thread", "value": "83094-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}